### ISSUE TITLE
Row, Column and Positioned widgets always have small gap when width/height is not integer

### ISSUE THREAD

**Issue by ph55** [flutter/flutter#46604]
Issue happens both on Android and IOS.

* Column:

```dart
// ...
        child: Column(
          children: <Widget>[
            Container(
              width: 100.5,
              height: 100.5,
              color: Colors.lightGreen,
            ),
            Container(
              width: 100.5,
              height: 100.5,
              color: Colors.lightGreen,
            )
          ],
        ),
```

Will result with (notice tiny gap between elements): 
![image](https://user-images.githubusercontent.com/1386622/70452004-101a9d00-1aaf-11ea-8567-608111f7e032.png)
![image](https://user-images.githubusercontent.com/1386622/70452055-258fc700-1aaf-11ea-82ca-84bced664d75.png)

* Row:

```dart
// ...
        child: Row(
          children: <Widget>[
            Container(
              width: 100.5,
              height: 100.5,
              color: Colors.lightGreen,
            ),
            Container(
              width: 100.5,
              height: 100.5,
              color: Colors.lightGreen,
            )
          ],
        ),
```

![image](https://user-images.githubusercontent.com/1386622/70452155-540da200-1aaf-11ea-9ea5-bd87453fd67f.png)

* Positioned:

```dart
// ...
        child: Stack(
          children: <Widget>[
            Positioned(
              top: 100,
              left: 0,
              width: 100.5,
              height: 100,
              child: Container(
                color: Colors.lightGreen,
              ),
            ),
            Positioned(
              top: 100,
              left: 100.5,
              width: 100.5,
              height: 100,
              child: Container(
                color: Colors.lightGreen,
              ),
            )
          ],
        ),
```

![image](https://user-images.githubusercontent.com/1386622/70452573-03e30f80-1ab0-11ea-8f39-a2cc28720bfd.png)

**Comment by HansMuller** [comment]
This looks like a known problem with antialiasing and mapping from logical pixels to device pixels. There's an open issue for a documentation fix: #14288

More info about the problem can be found here:
https://github.com/flutter/flutter/issues/14288#issuecomment-361375791
https://github.com/flutter/flutter/issues/17084#issuecomment-385718108
https://github.com/flutter/flutter/issues/15035#issuecomment-370028740

**Comment by ph55** [comment]
@HansMuller thanks, closing this as duplicate.
